lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -1,137 +1,116 @@
-// Adds combination of margin and padding to headings to make them appear
-// correctly when they're linked to as anchors, and scale them on mobile
-// TODO: Headings are hidden underneath the sticky 'table of contents' on mobile
-// when following a link to them
-
-@mixin heading-offset($tabletTopMargin) {
-  // Scale margins with font size on mobile (16/19ths)
-  $mobileTopMargin: ceil($tabletTopMargin * (16 / 19));
-
-  padding-top: min($mobileTopMargin, $gutter-half);
-  margin-top: max(0, $mobileTopMargin - $gutter-half);
-
-  @include media(tablet) {
-    padding-top: min($tabletTopMargin, $gutter);
-    margin-top: max(0, $tabletTopMargin - $gutter);
-  }
-}
-
 .technical-documentation {
+  @include govuk-font($size: 19);
   // Required for IE11 to render <main> correctly
   display: block;
-  margin: 0 $gutter-half 10px;
+  margin: govuk-spacing(3);
+  font-size: 19px;
   max-width: 40em;
 
   line-height: 1.4;
 
-  color: $text-colour;
+  color:  $govuk-text-colour;
 
-  @include media(tablet) {
+  @include govuk-media-query(tablet) {
     // We add 2 pixels to the gutter here to ensure there's sufficient spacing between
     // an anchor link (see _anchored-heading.scss) on a heading and the left-hand pane.
-    margin: 0 ($gutter + 2px) $gutter;
+    margin: 0 govuk-spacing(6) (govuk-spacing(6) + 2px);
   }
 
   .has-search-results-open & {
     visibility: hidden;
   }
 
-  // Headings
-  h1 {
-    @include bold-48;
-    @include heading-offset($gutter * 2);
-    border-top: 5px solid $text-colour;
+  > h1 {
+    @extend %govuk-heading-xl;
 
     &:first-of-type {
-      @include heading-offset($gutter);
-      border-top: none;
+      padding-top: govuk-spacing(6);
     }
   }
 
-  h2 {
-    @include bold-36;
-    @include heading-offset($gutter * 1.5);
+  > h2 {
+    @extend %govuk-heading-l;
+    padding-top: govuk-spacing(6);
+    margin-top: govuk-spacing(3);
   }
 
-  h3, h4, h5, h6 {
-    @include heading-offset($gutter);
+  > h3 {
+    @extend %govuk-heading-m;
   }
 
-  h3 {
-    @include bold-24;
+  > h4 {
+    @extend %govuk-heading-s;
   }
 
-  h4 {
-    @include bold-19;
+  > p {
+    @extend %govuk-body-m;
   }
 
-  h5 {
-    @include bold-19;
-    color: $secondary-text-colour;
+  > strong,
+  > b {
+    @include govuk-typography-weight-bold;
   }
 
-  h6 {
-    @include core-19;
-    color: $secondary-text-colour;
+  > ul,
+  > ol {
+    @extend %govuk-list;
   }
 
-  // Expand margin when h1 followed directly by paragraph
-  h1 + p {
-    @include media(tablet) {
-      margin-top: $gutter * 1.5;
-    }
+  > ol {
+    @extend %govuk-list--number;
   }
 
-  // Tighten up margins where headings follow each other
-  h1 + h2 {
-    @include heading-offset($gutter);
+  > ul {
+    @extend %govuk-list--bullet;
   }
 
-  h2 + h3, h3 + h4, h4 + h5, h5 + h6 {
-    @include heading-offset(20px);
+  a {
+    @extend %govuk-link;
```

Comment:
have you considered having the markup generators output the `.govuk-link` class with every link? I'm wondering if that would save us some work at all.

Reply:
I haven’t really considered anything but deffo all the CSS in this file is hacky. I think we just roll with it and yeah work on something better later. 

It deffo be nicer if got the markdown processor to add govuk classes

Reply:
I'd started doing it in some branch, it's definitely do-able but let's worry about it later.